addons/dogtags/stringtable.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Dogtags">
+        <Key ID="STR_ACE_Dogtags_itemName">
+            <English>Dog Tag</English>
+            <German></German>
+            <French></French>
+            <Czech></Czech>
+            <Polish>Nieśmiertelnik</Polish>
+            <Spanish></Spanish>
+            <Hungarian></Hungarian>
+            <Russian></Russian>
+            <Italian></Italian>
+            <Portuguese></Portuguese>
```

Comment:
String table entries cannot be empty for other languages because they will
display blank name for given language then. Either fill up rest of
languages or delete blank entries and let translators do their job.
Missed that functionality too.
19.03.2016 6:09 PM "jonpas" <notifications@github.com> napisał(a):

> In addons/dogtags/stringtable.xml
> <https://github.com/acemod/ACE3/pull/3600#discussion_r56751747>:
>
> > @@ -0,0 +1,89 @@
> > +<?xml version="1.0" encoding="utf-8"?>
> > +<Project name="ACE">
> > +    <Package name="Dogtags">
> > +        <Key ID="STR_ACE_Dogtags_itemName">
> > +            <English>Dog Tag</English>
> > +            <German></German>
> > +            <French></French>
> > +            <Czech></Czech>
> > +            <Polish>Nieśmiertelnik</Polish>
> > +            <Spanish></Spanish>
> > +            <Hungarian></Hungarian>
> > +            <Russian></Russian>
> > +            <Italian></Italian>
> > +            <Portuguese></Portuguese>
>
> Those shouldn't be here, just remove the empty ones.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/acemod/ACE3/pull/3600/files/ca3171a33a155563346f71e32d1d072a9447407f#r56751747>
>